app/components/post_textbox/post_textbox.js

```diff
@@ -96,6 +102,66 @@ export default class PostTextbox extends PureComponent {
         this.autocomplete = c;
     };
 
+    attachFileFromCamera = async () => {
+        try {
+            this.props.actions.closeModal();
+            const image = await ImagePicker.openCamera({
+                compressImageQuality: 0.5
+            });
+
+            this.uploadFiles([image]);
+        } catch (error) {
+            // If user cancels it's considered
+            // an error and we have to catch it.
+        }
+    }
+
+    attachFileFromLibrary = async () => {
+        try {
+            this.props.actions.closeModal();
+            const images = await ImagePicker.openPicker({
+                multiple: true,
+                compressImageQuality: 0.5
+            });
+
+            this.uploadFiles(images);
```

Comment:
It's worked really well for me on both platforms so far. Any issue with letting it be until we hear otherwise?